db/dcrpg/internal/addrstmts.go

```diff
@@ -67,7 +70,7 @@ const (
 	DeindexAddressTableOnAddress = `DROP INDEX uix_addresses_address;`
 
 	IndexAddressTableOnVoutID = `CREATE UNIQUE INDEX uix_addresses_vout_id
-		ON addresses(vout_row_id);`
+		ON addresses(vout_row_id, address);`
```

Comment:
Included address in this joint index, and update ON CONFLICT in insert statements to match